### DISCUSSION TITLE
1 new EiB in 24h?

### DISCUSSION THREAD

**Discussion by eduardocorral** [Chia-Network/chia-blockchain#5759] [Q&A] [answered]
I checked the total space yesterday (May 20), and it was around 7 EiB. We're at 8 EiB today.

Does this mean 1 EiB has been added in ~24h? 1 EiB = 1000 PiB = 1e6 TiB ? 100k 10TB disks? 

Is this possible? Is this estimation documented anywhere?

**Comment by T4C-FanBoy** [comment]
No wonder why I made 0$ so far :D

**Comment by AyeBraine** [comment]
This growth includes plotting. The total netspace is the amount of space which is live and farming. Also, May 20 and May 21 figures (at the same time of day) are 800 PiB apart. Anyway, 800 000 TB is quite a lot, but maybe that's what everyone plotting at once looks like.

There are 450 000 nodes that ever received Chia at least once. Judging by very active small farmer adoption, it's probable that there is much more nodes currently online, that haven't yet received any Chia. Even if the number is only twice as big, each participant on average would need to plot out 1 TB in that day.

**Reply by eduardocorral** [reply]
> This growth includes plotting.

Where can I read about this? Seems counterintuitive to include chunks of unprepared land in the final number.

**Reply by AyeBraine** [reply]
Frankly, I didn't understand your comment. But if I took a guess, I'm saying the opposite of what you wrote. I said that netspace figure is only the plotted space. The network can't magically guess how much space is on disks that farmers own.

"Growth includes plotting" means that when netspace grows, this not only means that someone bought or installed more disks, but that people also created more plots on disks that are already there. So in other words, it's not only "someone bought a ton of new disks", but rather "almost everyone on the network made new plots today".

Then, I do a rough calculation how fast this plotting process is. It's very reasonable. I already filled my current disks, but when I plotted, I did ~2 TB per day. And this is a tiny small 1 PC farm.

**Reply by eduardocorral** [reply]
>  I did ~2 TB per day

Sounds sketchy to me. 

https://www.chia.net/2021/02/22/plotting-basics.html

> A k32 plot can be done by one expert we know in just under 4 hours, but most experts are creating plots in 5 hours and most folks average around 9-12 hours.

To get to 2TB/day, in a single machine, you'd need to use proper HW, like RAID0 multilane PCIe NVMe enclosures, if being a home gamer, or fibre channel, or similar. So either it's not a _tiny small 1 PC farm_ (and this is plotting, not farming), or you made that up.

> This growth includes plotting. The total netspace is the amount of space which is live and farming.

Plotting is not farming. Farming is done on finished plots. Plotting is preparing the plots, but until they're done, they're invisible to the network. The only growth is the one that comes from adding finished plots to the network. So I really don't know what you mean with _growth includes plotting_. If you mean that finished plots contribute to the network, well, yes, that's the point. If you mean that all those temp files during the plotting process contribute to the network, you're wrong.

And that's why the original question: 

- we're saying that not only we're adding the equivalent of 50-100k 10TB drives to the network **each day**
- is that, for that to happen, we're saying that **we're managing to plot the equivalent of 50-100k 10TB each day**

Assuming a .5EiB/day growth, 100GiB plots, that's 5 million new plots a day. If there are 100k machines plotting 24/7, that's 50 plots/machine on average, which is impressive.

**Comment by thewhitegrizzli** [comment]
What's the point here. Of course it's possible. And it's why the project won't work, as well. It's one of the many reasons/causes/consequences of itself (the project and the flaws it has).
I think Chia will have paved the way for the Proof-of-Space-and-Time coins, but was clearly not ready for launch early May.
I cancelled hundreds of Terabytes I had ordered tonight. 
I no longer believe in that project.

**Reply by eduardocorral** [reply]
> Of course it's possible

Check the other answers, with some rough calculations. These numbers are such that skepticism should be expected. This is growth that shames the largest cloud vendors. Granted, probably temporal, the gold rush. Granted, this is what decentralization/pooling looks like. Still, .5-1.0EB/day is mind boggling. For comparison:

https://home.cern/science/computing/storage

The LHC produces ~115PB **a year** in total.

**Comment by thewhitegrizzli** [comment]
> I checked the total space yesterday (May 20), and it was around 7 EiB. We're at 8 EiB today.
> 
> Does this mean 1 EiB has been added in ~24h? 1 EiB = 1000 PiB = 1e6 TiB ? 100k 10TB disks?
> 
> Is this possible? Is this estimation documented anywhere?

yes that is exactly what it means and it is a VERY CLEAR red flag for small farmers like us/you/me.

**Comment by MaximusChia** [comment]
At some point this will collapse this growth wont be infinite nor exponential everyday, so not lucky small farmers could get a chance.
Other option is pooling, So will get a piece of reward with more frecuency

**Comment by room101-dev** [comment]
> I checked the total space yesterday (May 20), and it was around 7 EiB. We're at 8 EiB today.
> 
> Does this mean 1 EiB has been added in ~24h? 1 EiB = 1000 PiB = 1e6 TiB ? 100k 10TB disks?
> 
> Is this possible? Is this estimation documented anywhere?

That's about right yesterday am EB 7.1, late afternoon 7.4 this am 8.1, growth has always been 8% day, now nearing 10eb, that mean its grow 0.8EB a day, and it certainly is, and at 8% that means it doubles every nine days, which means your profit is going down by 1/2 every 9 days.

Lastly, if you ain't earning $$ on HPOOL since months ago, well you deserve zero, you should certainly know that all the dev's here and whales are on hpool, that's why they get 99% of all xch because hpool owns chia.

1 plot pays 7 cents a day right now, 16 plots $1,  160 plots $10/day, 1600 plots $100/day. It takes 400 plots to earn 1 XCH in one month.

There are 1,000's of idiots swallowing the chia bile about 'waiting for the pool protocol', in the meantime they're earning $$ and you ain't.

Yes, I did the calc's today, it means that daily right now they're adding 51k plots daily to the pool ( space ), I know I generate 16/day, I think there's lots of plot generation happening on amazon aws, until people get the bill, and realize a 'plot' only pays 7 cents a day ,and loses 1/2 the return every 9 days.
Solo-Mining quit paying +2 months ago, I know I have been doing both for boths, its a two-fer when you run on HPOOL, I'm running the CHIA-Harvester where I've never won, an on HPOOL I win ever 5-15 minutes.

Probably why dev crew here hates hpool is they let you double mine, you can be sure the pool-protocol will not let you double mine, in fact we already know that with the 30 min cool off period.

IMHO most of us will stay on HPOOL forever collect daily XCH, and still run solo just for humor, and there is nothing that chia-dev can do about it.

**Reply by AyeBraine** [reply]
What a large comment with a huge number of dubious statements.

I'll pick one: 2+ months ago, mainnet haven't yet launched. It happened only 2 months and 3 days ago. This means that Chia never paid at all.

Oh, and a small bonus track: 51 000 plots a day is 5150 TiB. Which is 0.65% of daily growth right now.

Please don't answer, if possible.

**Reply by eduardocorral** [reply]
> 0.8EB a day

That still doesn't add up. I think total SSD drive sales a year must be around ~300 million. Assuming 1TB per drive, that's 300EB, which is roughly .8EB/day. You're telling me that chia is adding the equivalent of every new sold SSD to the network. 

> Lastly, if you ain't earning $$ on HPOOL since months ago, well you deserve zero,

I don't care about the money. I'm doing this for fun and to learn. So calm your tits.

> you should certainly know that all the dev's here and whales are on hpool

I did not know, because I have a real job. Now I know, thanks.

> There are 1,000's of idiots swallowing the chia bile about 'waiting for the pool protocol', in the meantime they're earning $$ and you ain't.

I don't care. But now I get why everyone hates cryptominers, and it's not because they're all buying Lambos.

As I said, not for the money. There's more solid gains in the stock market. Check storage tech companies a year from now.

> Probably why dev crew here hates hpool

Checked hpool. Closed source binaries coming from China, in machines with an open port to the internet. Boy, what could go wrong. Not touching that with a 10ft pole. The CCP will have a field day siphoning out every bit they can get, before bringing down the hammer and creating the cheapest cloud storage vendor with the spoils

https://www.reuters.com/technology/chinese-financial-payment-bodies-barred-cryptocurrency-business-2021-05-18/

If you ask me, and I don't know the chia devs, I'd say they'd be more pissed at the fact that they started something they though could revolutionize crypto, only to be abused by the usual suspects, like waves of assholes ravaging an all-you-can-eat buffet.

> there is nothing that chia-dev can do about it

Wouldn't be so sure about that.

**Comment by T4C-FanBoy** [comment]
Plots in 4 hours as an expert? Dang I must be such a boss doing one plot every 90 mins.

**Reply by eduardocorral** [reply]
I tested multiple configs, the PCIe 16x with a RAID0 1TB drive as temp being obviously much better than the SSD RAID0 drives (theoretical 4x speed expected, got 3-3.5x). And yes, doing **1 plot** in less than 4h seems doable at this point. But plotting in parallel is another thing when you're loading the drives too much. Diminishing returns. Plotting in 90m means 18 plots a day, serial. Assuming you can plot, consistently, 4 in parallel in 90m, rounding it up, that's 100 plots, 10TB a day. Doable with a home gamer setup? No way. And we're adding 100k of those **each day**.

**Reply by Cactusbonus123** [reply]
Only theoretically...  What hapened if someone figured out how to hack the ploting mechanism to think the os and the client the plot is 101GB but only 1GB real size in it. This case on the 10TB HDD 100x data on it.

**Reply by eduardocorral** [reply]
Precisely. I hope it's not that simple, but that's why I'm asking these questions. Is is truly a matter of optimizing, or has someone found a flaw, and is exploiting it?

PoW needs to go, and we need better alternatives. But not flawed ones.

**Comment by eduardocorral** [comment, marked as answer]
I'll answer myself with a decent source, I hope

https://www.chiaexplorer.com/charts/netspace

**Comment by GeorgePapageorgakis** [comment]
Seems that small farmers are fakd up...

**Comment by eduardocorral** [comment]
https://www.newscientist.com/article/2278696-bitcoin-rival-chia-destroyed-hard-disc-supply-chains-says-its-boss/

> The resulting increase in demand has caused significant price rises for hard discs, especially higher-end models. The share price of hard disc maker Western Digital has increased from $52 at the start of the year to $73, while competitor Seagate is up from $60 to $94 over the same period.
